### DISCUSSION TITLE
deploy milvus cluster using docker-compose

### DISCUSSION THREAD

**Discussion by umitkarakas123** [milvus-io/milvus#17655] [Q&A and General discussion]
Hello i have 3 vm's for my milvus cluster. I want to deploy milvus cluster using docker-compose. but ı couldn't find any document or information. I hope you can help me.

here is my docker-compose.yml and milvus.yaml files; 

[milvus.yaml.txt](https://github.com/milvus-io/milvus/files/8941027/milvus.yaml.txt)
[docker-compose.yml.txt](https://github.com/milvus-io/milvus/files/8941028/docker-compose.yml.txt)

**Comment by engrabdu** [comment]
Hello Umitkarakkas...did you deploye the milvus cluster or not ?
